introduction.md

```diff
@@ -0,0 +1,144 @@
+
+The flexibility brought by the active defense mechanism allows any unvault or spending policy to be set and enforced by watchtowers. This therefore largely extends the scope of the possibilities compared to on-chain enforcement and is more in-phase with custodians’ needs.
+
+
+## Threat Model
+
+While Revault has been designed to fix (some of) the threats on existing custody protocols, it introduces new challenges and different assumptions. Revault aims to prevent theft with up to N-1 stakeholders' signing devices being compromised. Here we highlight different attack scenarios, trying to identify weaknesses in the design and finish with a set of security assumptions introduced when using Revault. For more detailed research, check the [Operational Risk Framework](https://github.com/revault/research/blob/master/risk_analysis/paper.pdf).
+
+### Emergency Deterrent Risk
+
+While the Emergency TX is the strongest mitigation in the Revault architecture, it introduces a potential risk of blackmail or losses derived from lack of business continuity (e.g. a block-and-short attack). An attacker (internal or external) getting hold of a single Emergency TX can threaten to push it to the Bitcoin network, triggering a push of all Emergency TXs by the watchtowers. The funds are not at risk in this scenario but getting the funds back online would be long and difficult. On the other hand, an important part of the security model (the deterrent)   is enforced by the pre-signed Emergency TXs being accessible.
+
+### Delegation Risk
+
+An "active defence" mechanism is used to restrict the behaviour of fund managers. Contrary to commonly deployed multi-signature wallets, it takes managers two transactions to spend funds. The first step encodes an on-chain time-lock. During this time, only a single honest watchtower is required to cancel given a breach of the unvault policy. It is expected and encouraged to deploy many watchtowers (at least one per stakeholder). 
+
+To by-pass the spend policy enforcement, an attacker would need to compromise all co-signing servers or to compromise all watchtowers. In the former case, an attacker could alter the Spend TX after the Unvault time-lock expires. In the latter case, a malicious Spend TX would not trigger the broadcast of a Cancel TX.
+
+### Denial-of-service Risk
+
+As a consequence of the N-1 resilience model, there are multiple attacks against the practical usage of the funds:
+
+- signing refusal by a stakeholder
+- unjustified Cancel TX push
+- unjustified Emergency TX push
+- taking down a co-signing server
+- taking down the coordinator
+
+In addition, if the network becomes highly congested, high feerates would trigger nodes to increase their minimum accepted feerate. Unvault TXs that are prepared with an initial 253 sat/kWU feerate could fail to be propagated sufficiently. Managers would lose access to funds delegated to them until stakeholders can re-sign Unvault TXs with a higher feerate. 
```

Comment:
```suggestion
In addition, if the network becomes highly congested, high feerates would trigger most [0] Bitcoin network nodes' to increase their mempool's minimum accepted feerate. Unvault TXs that are prepared with an initial 24 sat/vb feerate could fail to be relayed should the minimum accepted feerate increase sufficiently [1]. Managers would not be able to use funds delegated to them until the backlog of transactions is processed by the Bitcoin network (mined). 

[0] Those who didn't manually increase the already pretty-high mempool limits.
[1] It's exponential, so that's very likely that in case of the mempool getting above 300MB it quickly won't accept the Unvaults at 24sat/vb
```

It's been more than a year that we don't pre-sign Unvault transaction at 1sat/vbyte..